mne/viz/topomap.py

```diff
@@ -756,6 +756,8 @@ def plot_topomap(data, pos, vmin=None, vmax=None, cmap=None, sensors=True,
     %(topomap_sphere)s
     %(topomap_border)s
     %(topomap_ch_type)s
+    cnorm : matplotlib.colors.Normalize | None
+        Colormap normalization, default None means linear normalization.
```

Comment:
I would add a "see notes" here, and add to the notes section a link to MPL's docs or tutorial explaining what this is / how it works.  This is based on an impression / guess that most users won't know what "colormap normalization" means and thus won't be confident that the default is in fact what they want.